tests/e2e/profile-happy-path.test.ts

```diff
@@ -85,29 +87,29 @@ describe.only("Profile happy path", () => {
         );
 
         await click(localizedStrings.newProfile.add);
-        expect(await currentURL()).toEqual(profileUrl);
```

Comment:
Why did you remove this except ? It was meant to verify that the add profile flow worked as expected, since if it is successful, we are redirected to the Profile overview.

Reply:
It is useless in fact, because if we aren't on that page, the line 👇  will fail and will also make the test fail.
```ts
await click(localizedStrings.profileOverview.updateInfo);
```

At first I forgot to put it back but it doesn't matter so I didn't put it back 😁